package.json

```diff
@@ -36,99 +35,63 @@
     "babel-core": "6.26.0",
     "babel-eslint": "8.0.1",
     "babel-jest": "21.2.0",
-    "babel-loader": "7.1.2",
     "babel-plugin-array-includes": "2.0.3",
+    "babel-plugin-external-helpers": "^6.22.0",
     "babel-plugin-module-resolver": "2.7.1",
     "babel-plugin-transform-export-extensions": "6.22.0",
     "babel-plugin-transform-flow-strip-types": "6.22.0",
     "babel-plugin-transform-object-rest-spread": "6.26.0",
-    "babel-preset-env": "1.6.0",
+    "babel-preset-env": "1.6.1",
+    "babel-preset-stage-0": "^6.24.1",
     "check-node-version": "2.1.0",
-    "codecov": "2.3.0",
+    "codecov": "3.0.0",
     "commitizen": "2.9.6",
     "common-tags": "^1.4.0",
-    "cross-env": "5.0.5",
+    "cross-env": "5.1.0",
     "cz-lerna-changelog": "1.2.1",
-    "eslint": "3.19.0",
-    "eslint-config-airbnb": "15.1.0",
-    "eslint-config-commercetools": "6.0.0",
+    "eslint": "4.9.0",
+    "eslint-config-airbnb": "16.1.0",
+    "eslint-config-airbnb-base": "12.1.0",
+    "eslint-config-prettier": "^2.6.0",
     "eslint-formatter-pretty": "1.3.0",
     "eslint-import-resolver-babel-module": "3.0.0",
     "eslint-plugin-flowtype": "2.39.1",
-    "eslint-plugin-import": "2.7.0",
-    "eslint-plugin-jsx-a11y": "5.1.0",
+    "eslint-plugin-import": "2.8.0",
+    "eslint-plugin-jest": "^21.2.0",
+    "eslint-plugin-jsx-a11y": "6.0.2",
+    "eslint-plugin-prettier": "^2.3.1",
     "eslint-plugin-react": "7.4.0",
-    "flow-bin": "0.56.0",
+    "flow-bin": "0.57.3",
     "gitbook-cli": "2.3.2",
     "gitbook-plugin-anchorjs": "^1.1.1",
     "gitbook-plugin-edit-link": "^2.0.2",
     "gitbook-plugin-ga": "^1.0.1",
     "gitbook-plugin-github": "^2.0.0",
     "gitbook-plugin-prism": "^2.2.0",
-    "husky": "0.14.0",
+    "husky": "0.14.3",
     "jest": "21.2.1",
     "lerna": "2.4.0",
     "lerna-semantic-release": "9.1.0",
     "npm-check-updates": "2.13.0",
     "npm-run-all": "4.1.1",
+    "prettier": "^1.7.4",
     "rimraf": "2.6.2",
+    "rollup": "^0.50.0",
+    "rollup-plugin-babel": "^3.0.2",
+    "rollup-plugin-commonjs": "^8.2.1",
+    "rollup-plugin-filesize": "^1.4.2",
+    "rollup-plugin-includepaths": "^0.2.2",
+    "rollup-plugin-json": "^2.3.0",
+    "rollup-plugin-node-resolve": "^3.0.0",
+    "rollup-plugin-replace": "^2.0.0",
+    "rollup-plugin-uglify": "^2.0.1",
+    "rollup-watch": "^4.3.1",
     "streamtest": "1.2.2",
-    "validate-commit-msg": "2.14.0",
-    "webpack": "3.6.0"
+    "validate-commit-msg": "2.14.0"
   },
   "config": {
     "commitizen": {
       "path": "./node_modules/cz-lerna-changelog"
     }
-  },
-  "eslintConfig": {
-    "extends": "commercetools",
```

Comment:
I noticed that this is not being used anymore in merchant center hence the difference coding convention in different teams, but now that we are moving more towards smaller teams and hopefully smaller repos. I think we need to maintain this so we have a consistent coding style, across the company.

AFAIK, we are the only team using this config and if we remove it now then the config itself will be useless. I think we should all agree on a coding style and maintain it across the company.

Cc @emmenko